packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -131,7 +131,7 @@ export class ProfilesCache {
     /**
      * Used for extenders to register with Zowe Explorer that do not need their
      * profile type in the existing MVS, USS, and JES
-     *
+     * @deprecated Please use asyncRegisterCustomProfilesType
```

Comment:
```ts
    /**
     * @deprecated Please use {@link asyncRegisterCustomProfilesType}.
     * 
     * Used for extenders to register with Zowe Explorer that do not need their
     * profile type in the existing MVS, USS, and JES
     *
     * @param {string} profileTypeName Type of Profile, defaults to "zosmf" if nothing passed.
     *
     * @returns {void}
     */
```

Minor: a proposal to use a link for the new function and move `@deprecated` annotation to the top of the JSDoc.